modules/web/package.json

```diff
@@ -60,28 +60,30 @@
     "socks": "2.7.3"
   },
   "dependencies": {
-    "@angular/animations": "17.3.1",
-    "@angular/cdk": "17.3.1",
-    "@angular/common": "17.3.1",
-    "@angular/compiler": "17.3.1",
-    "@angular/core": "17.3.1",
+    "@angular/animations": "17.3.5",
+    "@angular/cdk": "17.3.5",
+    "@angular/common": "17.3.5",
+    "@angular/compiler": "17.3.5",
+    "@angular/core": "17.3.5",
     "@angular/flex-layout": "15.0.0-beta.42",
-    "@angular/forms": "17.3.1",
-    "@angular/localize": "17.3.1",
-    "@angular/material": "17.3.1",
-    "@angular/platform-browser": "17.3.1",
-    "@angular/platform-browser-dynamic": "17.3.1",
-    "@angular/router": "17.3.1",
+    "@angular/forms": "17.3.5",
+    "@angular/localize": "17.3.5",
+    "@angular/material": "17.3.5",
+    "@angular/platform-browser": "17.3.5",
+    "@angular/platform-browser-dynamic": "17.3.5",
+    "@angular/router": "17.3.5",
     "@fontsource/inconsolata": "4.5.10",
-    "@fontsource/roboto": "5.0.12",
-    "@fontsource/roboto-mono": "5.0.17",
+    "@fontsource/roboto": "5.0.13",
+    "@fontsource/roboto-mono": "5.0.18",
     "@fontsource/ubuntu": "4.5.11",
     "@swimlane/ngx-charts": "20.5.0",
+    "@types/react": "^18.2.79",
+    "@types/react-dom": "^18.2.25",
```

Comment:
Should we add them to `devDependencies`? Since these are not used at runtime.

Reply:
Updated 👍